### PR TITLE
[Ready] Improved mission clean up logic and implemented related opcodes

### PR DESCRIPTION
**Pull request by dracc**
As written by @Husho in #412

### CODE REVIEW

**Comment by dracc** [comment]
LGTM.
No weird comments and the logic seems, well, logical.

File: rwengine/src/script/modules/GTA3ModuleImpl.inl [hunk 1]
```diff
@@ -10696,23 +10728,27 @@ void opcode_0394(const ScriptArguments& args, const ScriptInt arg1) {
     @arg clearParticles Boolean true/false
 */
 void opcode_0395(const ScriptArguments& args, ScriptVec3 coord, const ScriptFloat radius, const ScriptBoolean clearParticles) {
-    GameWorld* gw = args.getWorld();
+    auto gw = args.getWorld();
```
Comment (husho) [inline rwengine/src/script/modules/GTA3ModuleImpl.inl]: this should be a pointer

Comment (dracc) [reply rwengine/src/script/modules/GTA3ModuleImpl.inl]: Seems to me I ought to revert it back to `GameWorld* gw`. Do you agree?

Comment (husho) [reply rwengine/src/script/modules/GTA3ModuleImpl.inl]: yes

**Comment by dracc** [comment]
Opcode 02dd looks like it was removed and rewritten with the same code. Reason for this is that it mixed tabs and spaces and looked horrid when using other tab-width than the original authors.
I'm sure there are more instances of this, but I stumbled upon this one and figured I "might as well".

**Comment by dracc** [comment]
Hmm... This one's a little odd.
<a href="https://i.imgur.com/SZJCH9p.png">
  
</a>

**Pull request merged**